test/test02/cmd/aux/main.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
So these are setting up a series of subdirectories that are actually each an entire properly-structured go project, to test out commands on?

Reply:
Yes.  There is no unit testing suite for Makefiles that I'm aware of.  So these are some small, trivial projects that use the Make infrastructure to test it.